src/components/ImageAlignmentPage.vue

```diff
@@ -290,6 +304,7 @@
        this.angle = 0;
        this.initialTransform = [[1,0,0],[0,1,0],[0,0,1]];
        this.alreadyUploaded = false;
+       document.querySelector('.input-optical-image').value='';
```

Comment:
I wonder why this line is needed?

Reply:
Otherwise the image won't be updated if you decide to upload it once again in the browser (chrome only). 
Here is more info: https://stackoverflow.com/questions/12030686/html-input-file-selection-event-not-firing-upon-selecting-the-same-file

Reply:
thanks for fixing this bug. I forgot to report it